<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysis Program</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="fileButton">
      <property name="text">
       <string>Choose file</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="fileLabel">
      <property name="font">
       <font>
        <family>Source Code Pro</family>
       </font>
      </property>
      <property name="text">
       <string>Please choose a file first.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="settingButton">
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Analyze</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="bugsCheckBox">
      <property name="text">
       <string>Only show bugs</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="font">
       <font>
        <family>Source Code Pro</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>685</width>
         <height>298</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>fileButton</tabstop>
  <tabstop>settingButton</tabstop>
  <tabstop>startButton</tabstop>
  <tabstop>bugsCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
